### ISSUE TITLE
Playwright tracing doesn't create page snapshots

### ISSUE THREAD

**Issue by iSuslov** [daijro/camoufox#101]
### Describe the bug:

When using playwright trace feature, resulting traces have no page snapshots.

### Screenshots:

This is how it looks when trace doesn't have page snapshots:

<img width="1101" alt="Screenshot 2024-11-25 at 12 21 00 AM" src="https://github.com/user-attachments/assets/8e38f28d-b0e1-44cf-b8de-0e8d4a7b5af0">

This is how it should look:
<img width="1101" alt="Screenshot 2024-11-25 at 12 22 18 AM" src="https://github.com/user-attachments/assets/35028528-0d6b-4eb3-a48c-8bf74253b824">

### To Reproduce:

```python
from camoufox.sync_api import Camoufox

with Camoufox(
    geoip=True,
    headless=True
) as browser:
    context = browser.new_context()
    context.tracing.start(screenshots=True, snapshots=True)
    page = context.new_page()
    page.goto("https://google.com")
    page.wait_for_timeout(4000)
    context.tracing.stop(path="trace.zip")
    context.close()
    page.close()
    browser.close()
```

### Version:

```console
Pip package:    v0.4.3
Camoufox:       v132.0.2-beta.17 (Up to date!)
```

### Additional info:

- changing headless to `True` / `False` / `Virtual` doesn't solve the problem
- no matter remote or local. Tested on Ubuntu 24.04 (remote, macos as a client) and MacOS 15.1 (local)
- traces have screenshots and even have html files inside, but `trace.trace` (inside archive) doesn't have these entities:

```json
{"type":"frame-snapshot","snapshot":{"callId":"call@8","snapshotName":"before@call@8","pageId":"page@b4ae9731aa790003d2fa59cc110a50ca","frameId":"frame@833b9e40f35a84590a34d52abd359c48","frameUrl":"about:blank","html":["HTML",{},["HEAD",{},["BASE",{"href":"about:blank"}]],["BODY"]],"viewport":{"width":1280,"height":720},"timestamp":1325.626,"wallTime":1732510550491,"collectionTime":2,"resourceOverrides":[],"isMainFrame":true}}
```
### Difference in size:

"Bad" trace:
<img width="530" alt="Screenshot 2024-11-25 at 12 32 18 AM" src="https://github.com/user-attachments/assets/63d9e492-e89f-402d-95f1-27fc290da074">

"Good" trace:
<img width="537" alt="Screenshot 2024-11-25 at 12 33 42 AM" src="https://github.com/user-attachments/assets/d67c9caf-4609-412b-afc3-eedf40f9b4ab">

**Comment by alex-dixon** [comment]
my guess is playwright needs to be set up with `tracesDir` set to something in the equivalent of browser.launch

**Comment by vk496** [comment]
I have the same issue. @iSuslov did you found any workaround?

**Comment by iSuslov** [comment]
@vk496 no, I just gave up on this assuming it is broken by custom camoufox patches, but it's just a speculation
